attributes/default.rb

```diff
@@ -1,7 +1,15 @@
 default['ganeti']['version'] = nil
 default['ganeti']['hypervisor'] = 'kvm'
+default['ganeti']['services'] = %w(
+  ganeti-confd
+  ganeti-kvmd
+  ganeti-luxid
+  ganeti-noded
+  ganeti-rapi
+  ganeti-wconfd
+)
 default['ganeti']['drbd'] = true
-default['ganeti']['master-node'] = nil
+default['ganeti']['master-node'] = false
```

Comment:
You might want to update the README, as it still says `nil` is the default value.